File: app/src/main/java/mx/edu/upqroo/kristenandroid/services/sie/messages/LoginMessage.java
package mx.edu.upqroo.kristenandroid.services.sie.messages;

import mx.edu.upqroo.kristenandroid.database.entities.UserInformation;

public class LoginMessage {
    private boolean result;
    private UserInformation student;
    private String message;

    public LoginMessage(boolean result, UserInformation student) {
        this.result = result;
        this.student = student;
    }

    public LoginMessage(boolean result, String message) {
        this.result = result;
        this.message = message;
    }


    public boolean isResult() {
        return result;
    }

    public void setResult(boolean result) {
        this.result = result;
    }

    public UserInformation getStudent() {
        return student;
    }

    public void setStudent(UserInformation student) {
        this.student = student;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/mx/edu/upqroo/kristenandroid/api/sie/messages/LoginMessage.kt
package mx.edu.upqroo.kristenandroid.api.sie.messages

import mx.edu.upqroo.kristenandroid.data.database.entities.UserInformation

class LoginMessage {
    var isResult: Boolean = false
    var student: UserInformation? = null
    lateinit var message: String

    constructor(result: Boolean, student: UserInformation) {
        this.isResult = result
        this.student = student
    }

    constructor(result: Boolean, message: String) {
        this.isResult = result
        this.message = message
    }
}
